### PR TITLE
Fix - video symlink play issue in safari in popup

### PR DESCRIPTION
**Pull request by chetansatasiya**
### All Submissions:

* [x] Have you followed the [Pull Requests Guidelines](https://github.com/buddyboss/buddyboss-platform/wiki/Submitting-Pull-Requests#pull-request-guidelines)?
* [x] Does your code follow the [WordPress' Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?

### Changes proposed in this Pull Request:

Fixes #ISSUE_NUMBER .

### How to test the changes in this Pull Request:

1.
2.
3.

### Other information:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your changes, as applicable?
* [x] Have you successfully run tests with your changes locally?

### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.

### CODE REVIEW

File: src/bp-core/bp-core-functions.php [hunk 1]
```diff
@@ -5690,7 +5691,12 @@ function bb_check_ios_device() {
 		$is_ios = true;
 	}
 
-	return $is_ios;
+	/**
+	 * Filter for the check if it's ios devices or not.
+     *
+     * @since BuddyBoss 1.7.1
```
Comment (KartikSuthar) [inline src/bp-core/bp-core-functions.php:5697]: @chetansatasiya updated since tag to 1.7.0.1

File: src/bp-video/bp-video-filters.php [hunk 2]
```diff
@@ -1671,3 +1674,28 @@ function bb_video_update_video_symlink( $response, $post_data ) {
...
 }
+
+/**
+ * Pass the true if the browser is safari and video need to play in popup.
+ *
+ * @param bool $is_ios whether a device is a ios.
+ *
+ * @return bool|mixed
+ *
+ * @since BuddyBoss 1.7.1
```
Comment (KartikSuthar) [inline src/bp-video/bp-video-filters.php:1685]:
```suggestion
 * @since BuddyBoss 1.7.0.1
```

File: src/bp-video/bp-video-filters.php [hunk 3]
```diff
@@ -1671,3 +1674,28 @@ function bb_video_update_video_symlink( $response, $post_data ) {
...
+ */
+function bb_video_safari_popup_video_play( $is_ios ) {
+
+	if ( false === $is_ios ) {
+		$is_safari = stripos( $_SERVER['HTTP_USER_AGENT'], 'Safari' );
+		$action    = filter_input( INPUT_POST, 'action', FILTER_SANITIZE_STRING );
+
+		if ( $is_safari && 'video_get_activity' === $action ) {
+			$is_ios = true;
+        }
```
Comment (KartikSuthar) [inline src/bp-video/bp-video-filters.php:1695]: apply a phpcs here.

**Review by KartikSuthar — CHANGES_REQUESTED** [review]

File: src/bp-core/bp-core-functions.php (hunk 1 above)
Comment (chetansatasiya) [reply src/bp-core/bp-core-functions.php]: done

File: src/bp-video/bp-video-filters.php (hunk 2 above)
Comment (chetansatasiya) [reply src/bp-video/bp-video-filters.php]: done

File: src/bp-video/bp-video-filters.php (hunk 3 above)
Comment (chetansatasiya) [reply src/bp-video/bp-video-filters.php]: done

**Comment by chetansatasiya** [comment]
@KartikSuthar all changes applied.

**Review by KartikSuthar — DISMISSED** [review]
@chetansatasiya Please merge to dev and pass to QA.

**Comment by mohit-buddyboss** [comment]
@chetansatasiya @KartikSuthar 

FIXED> Tested on dev branch and 'fix/2553' of the platform. Tested on @chetansatasiya mac machine. Now videos are playing under safari in Pop UP. 

Video: https://www.loom.com/share/7930a5f6c03849c0aa785fb6fae9288c

Related issue is resolve now and PR is ready for release. 

cc @sourabhmatolia

**Pull request merged**